explainaboard/processors/kg_link_tail_prediction.py

```diff
@@ -124,6 +124,35 @@ def process(self, metadata: dict, sys_output: List[dict]) -> SysOutputInfo:
         self._user_defined_feature_config = metadata.get(
             "user_defined_features_configs", None
         )
+
+        # add user-defined features into features list
```

Comment:
Ideally this part should be put inside the base processor, but the way that default features for each task are initialized, it's hard to do that